src/index.css

```diff
@@ -177,125 +177,109 @@
     -webkit-text-fill-color: transparent;
   }
 
-  .h1_32_40_heavy {
+  .text_h1_heavy {
     @apply text-32px leading-10 font-extrabold;
   }
 
-  .h2_24_32_heavy {
+  .text_h2_heavy {
     @apply text-2xl font-extrabold;
   }
 
-  .h2_24_32_medium {
+  .text_h2_medium {
     @apply text-2xl font-medium;
   }
 
-  .h3_22_30_heavy {
+  .text_h3_heavy {
     @apply text-h3 leading-30px font-extrabold;
   }
 
-  .h4_18_24_heavy {
+  .text_h4_heavy {
     @apply text-h4 leading-6 font-extrabold;
   }
 
-  .h4_18_24_medium {
+  .text_h4_medium {
     @apply text-h4 leading-6 font-medium;
   }
 
-  .h4_18_24_roman {
+  .text_h4 {
     @apply text-h4 leading-6 font-normal;
   }
 
-  .h5_16_24_heavy {
+  .text_h4_leading_28_heavy {
```

Comment:
Could you remove classes with `leading`?
First because I can't see h4 with different line height in design
Second because better to have a standard class for h4, like `text-h4-heavy`, and then you can add other classes: `className='text-h4-heavy leading-[28px]'` in case if in some place of design it is different

Reply:
@CrazyOptimist 

No, we can't remove it. 
First --> It has some places that have been used the different line-height. See the attachment ( line-height.jpg) 
![line-height](https://user-images.githubusercontent.com/10944387/130562127-f4c0cdef-d1ac-4f27-8685-cfec65f80bf6.jpg) and I think it hasn't been defined in the typography yet.
Second --> We can't use `className='text-h4-heavy leading-[28px]'` to overwrite the line-height in the `text-h4-heavy`

Reply:
We can use `className='text-h4-heavy leading-[28px]'` if move text classes from `utilities` to `components` layer, so other utility classes will have precedence

Reply:
OK I will check and update in the next PR